doc/snippets/Microsoft.Data/SqlDbTypeExtensions.xml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0"?>
+<docs>
+  <members name="SqlDbTypeExtensions">
+    <SqlDbTypeExtensions>
+      <summary>Extensions for SqlDbType.</summary>
+      <remarks>
+        <format type="text/markdown">
+          <![CDATA[  
+
+## Remarks  
+The SqlDbTypeExtensions class provides new SqlDbType enums. These are meant to be used for applications which cannot leverage the enums in SqlDbType, which are available in newer .NET runtime versions.
+
+]]>
+        </format>
+      </remarks>
+    </SqlDbTypeExtensions>
+    <SqlJson name="default">
```

Comment:
Why is it named "default"?

Reply:
Inspired from SqlCommand.xml 
The name wont really matter here, since we are referencing the Xpath with the Tag name i.e. `SqlJson` in the Cs file. 
The name helps disambiguate between tags, which is not the case here.

Reply:
I suggest we remove it if it's not needed.